apps/wallet-dashboard/components/Dialogs/SendToken/SendTokenDialog.tsx

```diff
@@ -36,11 +36,7 @@ function SendTokenDialogBody({
 
     const { data: coinsData } = useGetAllCoins(selectedCoin.coinType, activeAddress);
 
-    const {
-        mutateAsync: signAndExecuteTransaction,
-        error,
-        isPending,
-    } = useSignAndExecuteTransaction();
+    const { mutateAsync: signAndExecuteTransaction, isPending } = useSignAndExecuteTransaction();
 
     const { data: transaction } = useSendCoinTransaction(
```

Comment:
To generate the tx, we need to share the `formData.formattedAmount` instead of the `formData.amount`. If not, we are sending the amount in nano